src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -185,6 +185,7 @@ TEST_F(CommandRequestImplTest, OnTimeOut_StateAwaitingHMIResponse_SUCCESS) {
 
   command->onTimeOut();
 
+  EXPECT_TRUE((*dummy_msg)[am::strings::msg_params][am::strings::info].asString().empty());
```

Comment:
@VVeremjova fixed in https://github.com/smartdevicelink/sdl_core/pull/1368/commits/e1e998ffc82c12f956a5d10777d9061aa13daae2